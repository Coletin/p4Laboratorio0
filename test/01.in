
1
Pepito
maig@gmail.com
1
1
Mauricio
mauri@gmail.com
1
2
401
20
2
5
maig@gmail.com
401
1
12112022
15112022
2
mauri@gmail.com
6
7
