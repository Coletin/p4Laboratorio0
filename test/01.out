1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

1>1.Agregar Huesped.
Ingrese el nombre(sin espacios)
Ingrese el email
Es Finger? 1-Si 0-No
1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

2>1.Agregar Huesped.
Ingrese el nombre(sin espacios)
Ingrese el email
Es Finger? 1-Si 0-No
1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

3>2.Agregar Habitacion.
Ingrese el numero
Ingrese el precio
Ingrese la capacidad
1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

4>5.Registrar Reserva.
Ingrese email del huesped principal: Habitacion: Ingrese codigo: Ingrese fecha de entrada (DD/MM/AAAA): stoi1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

5>Comando no reconocido.
1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

6>2.Agregar Habitacion.
Ingrese el numero
Ingrese el precio
Ingrese la capacidad
Ya existe una habitación con el número indicado1.Agregar Huesped.
2.Agregar Habitacion.
3.Obtener Huespedes.
4.Obtener Habitaciones.
5.Registrar Reserva.
6.Obtener Reserva.
7.Salir.

7>Fin.
7.Salir.
